examples/src/main/scala/org/apache/spark/examples/streaming/RecoverableNetworkWordCount.scala

```diff
@@ -116,7 +116,7 @@ object RecoverableNetworkWordCount {
     val lines = ssc.socketTextStream(ip, port)
     val words = lines.flatMap(_.split(" "))
     val wordCounts = words.map((_, 1)).reduceByKey(_ + _)
-    wordCounts.foreachRDD { (rdd, time) =>
+    wordCounts.foreachRDD { (rdd: RDD[(String, Int)], time: Time) =>
```

Comment:
FWIW I'm fine with putting this back, myself, but it's not consistent with all of the other examples -- streaming ones yes but not the non-streaming ones. Not a big deal. I think it's worth just getting this resolved.